integration_test.go

```diff
@@ -183,6 +188,49 @@ func TestRun(t *testing.T) {
 	}
 }
 
+func TestPidRun(t *testing.T) {
+	client, server := startCamusWithConfig(t, "testapp/deploy2.json")
+	defer server.Kill()
+	defer client.Shutdown()
+
+	client.Build()
+	deployId := client.Push()
+	port := client.Run(deployId)
+	data := getLocalhost(t, port, "")
+
+	//TODO(tim) nice way to test Enforce() in out-of-process server? :s
+	deploys := client.ListDeploys()
+	var node *Deploy
+	var haproxy *Deploy
+	for _, d := range deploys {
+		fmt.Printf(" - %s on %d @ %d - %s\n", d.Id, d.Port, d.Pid, d)
+		//deploy commit 
+		if regexp.MustCompile("\\d\\d\\d\\d-\\d\\d-\\d\\d-\\d\\d-\\d\\d-\\d\\d").MatchString(d.Id) {
+			if node != nil {
+				t.Fatalf("multiple node deploys running %s and %s", d.Id, node.Id)
+			}
+			node = d
+		} else if regexp.MustCompile("haproxy-\\d+").MatchString(d.Id) {
+			if haproxy != nil {
+				t.Fatalf("multiple haproxy instances running %s and %s", d.Id, haproxy.Id)
```

Comment:
can you reword "haproxy" to "testapp haproxy" because otherwise confusion with camus' haproxy instance